foxglove/cmd/export.go

```diff
@@ -942,6 +946,7 @@ func newExportCommand(params *baseParams) (*cobra.Command, error) {
 	exportCmd.PersistentFlags().StringVarP(&deviceName, "device-name", "", "", "device name")
 	exportCmd.PersistentFlags().StringVarP(&outputFile, "output-file", "o", "", "output file")
 	exportCmd.PersistentFlags().StringVarP(&recordingID, "recording-id", "", "", "recording ID")
+	exportCmd.PersistentFlags().StringVarP(&key, "key", "", "", "idempotency key")
```

Comment:
Is it clear to users what this means? Should we mention "recording" in some way?